cloud/cloud.go

```diff
@@ -288,18 +288,21 @@ func (c *Client) SyncDeploymentLogs(
 	stackID int64,
 	deploymentUUID UUID,
 	logs DeploymentLogs,
+	stackPreviewID string,
 ) error {
 	err := logs.Validate()
 	if err != nil {
 		return errors.E(err, "failed to prepare the request")
 	}
-	// Endpoint:/v1/stacks/{org_uuid}/{stack_id}/deployments/{deployment_uuid}/logs
-	_, err = Post[EmptyResponse](
-		ctx, c, logs,
-		c.URL(path.Join(
-			StacksPath, string(orgUUID), strconv.Itoa64(stackID), "deployments", string(deploymentUUID), "logs",
-		)),
-	)
+
+	url := c.URL(path.Join(
+		StacksPath, string(orgUUID), strconv.Itoa64(stackID), "deployments", string(deploymentUUID), "logs",
+	))
+	if previewMode := stackPreviewID != ""; previewMode {
+		url = c.URL(path.Join(StackPreviewsPath, string(orgUUID), stackPreviewID, "logs"))
```

Comment:
IMHO This is hacky and it may be misleading. we could reuse the same structure to send logs, changing the name for something like `CommandLogs` but `SyncDeploymentLogs` must not send preview logs

Reply:
Makes sense, `CommandLogs` sounds good to me. I will rename the associated functions/types.

Reply:
https://github.com/terramate-io/terramate/pull/1448